routers/web/repo/release.go

```diff
@@ -200,6 +206,51 @@ func releasesOrTags(ctx *context.Context, isTagList bool) {
 	ctx.HTML(http.StatusOK, tplReleases)
 }
 
+// tags render the tags list if repo's release is disabled.
+func tags(ctx *context.Context) {
```

Comment:
Why can't it reuse the `releasesOrTags` function? I guess most backend code/template code is the same?

Reply:
Yes, most of the code is the same. In the `tags` function I remove some unnecessary logic to simplify the code and improve performance.

Reply:
Why not reuse?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/23465/commits/87ea9ad84c450d547a80d9be3f44d5382b5262a8